src/main/java/nl/tudelft/cse1110/andy/result/ResultBuilder.java

```diff
@@ -303,6 +303,9 @@ private double calculateTimeItTookToRun() {
     }
 
     private int calculateFinalGrade(GradeValues grades, GradeWeight weights) {
+        if(grades.getTotalBranches() > 0 && grades.getCoveredBranches() == 0)
```

Comment:
These two conditions in the if could become a simple method call inside `GradeValues`, e.g., `if(grades.noBranchesCovered()` or something like that.

Reply:
Done